ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,326 @@
+
+    !!! warning
+        The playbook is set to "Run," and the EOS instances in use will be changed. Please ensure you are leveraging nonproduction instances when testing.
+
+    ![Template save](../_media/getting-started/aap-avd/template-save.png)
+
+### Surveys
+
+With most jobs, we need a way to authenticate to our CVP instance or EOS nodes. AAP provides a multitude of ways to define credentials. Some options are credentials for Network devices, Container registries, HashiCorp Vault, etc. Feel free to explore any option you need for your environment. For this guide, we will leverage a survey secret. Surveys allow us to ask users for information we require for a job to execute correctly. We will leverage a survey with a default secret value (this is the credential used to connect to our nodes).
+
+=== "Surveys"
+
+    On the center pane, click on `Survey`.
+
+    ![Click on Survey](../_media/getting-started/aap-avd/template-survey.png)
+
+=== "Surveys - Save"
+
+    - On the center pane, select `Add`.
+    - Set `Question` to an appropriate value.
+    - Set `Answer variable name` to any valid variable name, here we use `lab_pass`.
+    - `Answer type` is set to `Password` and the default value is set.
+    - Click `Save` when done.
+
+    ![Survey save](../_media/getting-started/aap-avd/survey-save.png)
+
+    The `lab_pass` variable or whichever name you would prefer can be used in your project by assigning the correct variable value to the `ansible_password` variable.
+
+    ```yaml
+    vars:
+      ansible_user: arista
+      ansible_password: "{{ lab_pass }}"
+      ansible_network_os: arista.eos.eos
+    ```
+
+=== "Surveys - Enable"
+
+    Click the `Surveys Disabled` radio button to ensure the survey is enabled.
+
+    ![Survey enable](../_media/getting-started/aap-avd/survey-enabled.png)
+
+### Running the Template with CVP
+
+Below is an example of the playbook we are leveraging to build and deploy our configurations with CVP. The following section will have an alternate version that can be deployed directly to EOS nodes.
+
+```yaml
+---
+- name: Manage Arista EOS EVPN/VXLAN Configuration
+  hosts: ATD_FABRIC
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.avd
+  vars:
+    fabric_dir_name: "{{ fabric_name }}"
+  tasks:
+
+    - name: Generate intended variables
+      import_role:
+        name: eos_designs
+
+    - name: Generate device intended config and documentation
+      import_role:
+        name: eos_cli_config_gen
+
+- name: Configuration deployment with CVP
+  hosts: cv_servers
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.avd
+  tasks:
+    - name: Provision CVP with AVD configuration
+      import_role:
+        name: eos_config_deploy_cvp
```

Comment:
Perhaps add a Studios example with cv_deploy?

Reply:
Do you mean to replace this one or show an additional build and deploy step with cv_deploy?

Reply:
I agree here, we should start giving examples for the role `cv_deploy`, considering it is the new way to use static configlets studios.

Reply:
Done, guide has been updated to leverage `cv_deploy`